flow/networks/pedestrian_traffic_light_grid.py

```diff
@@ -0,0 +1,644 @@
+"""Contains the traffic light grid scenario class."""
```

Comment:
Will probably want to merge this with the traffic light network

Reply:
fixed